src/Network/GRPC/MQTT/Types.hs

```diff
@@ -123,5 +142,5 @@ data ClientHandler where
 -- streamed in a short time. On the other hand, it is not a good idea to
 -- batch RPCs that send small messages infrequently (long-poll) because
 -- messages will not be published immediately.
-data Batched = Unbatched | Batched
-  deriving (Eq, Ord, Lift)
+-- data Batched = Unbatched | Batched
```

Comment:
I would just delete this code since you've replace it with the newtype + pattern synonyms.